<ui version="4.0" >
 <class>PlotSystemMenu</class>
 <widget class="QWidget" name="PlotSystemMenu" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>94</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Ignored" hsizetype="Ignored" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" >
   <item>
    <widget class="QCheckBox" name="autorangeCheckBox" >
     <property name="text" >
      <string>Auto Range</string>
     </property>
     <property name="shortcut" >
      <string>R</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="normalizeCheckBox" >
     <property name="text" >
      <string>Normalize histograms</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QRadioButton" name="variableButton" >
       <property name="text" >
        <string>Variable</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="squareButton" >
       <property name="text" >
        <string>Square</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rectButton" >
       <property name="text" >
        <string>Rect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
